server/src/config.rs

```diff
@@ -16,6 +16,14 @@ pub(crate) struct Context {
     pub(crate) conn_manager: Arc<DBConnectionManager>,
 }
 
+/*
+#[derive(Clone)]
+pub(crate) enum NetworkConfig {
+    Mainnet,
+    Devnet,
+}
+ */
+
```

Comment:
This would be missing the Berkeley testnet which is important

Reply:
Yes, added